framework/auth/views.py

```diff
@@ -245,13 +245,8 @@ def auth_login(auth, **kwargs):
         next_url = request.args.get('redirect_url')
         must_login_warning = False
 
-    if next_url:
```

Comment:
No, I'm wrong. The if is totally where it should be in that function. Ignore me.